### DISCUSSION TITLE
Negative frequencies and unstable structure

### DISCUSSION THREAD

**Discussion by RicardoCecconello** [ttadano/alamode#157] [General]
Dear Dr. Tadano,

I have an unstable structure and I'm calculating the harmonic IFCs with a supercell. It is known that there should be negative frequencies (from another work with the same structure, but a different method of getting the spectrum), but I'm not getting them. This happens for different values of displacement and cutoffs. For the smaller displacements, the fitting error is ~3%. Any idea why this is happening?

Thanks in advance,
Ricardo.

**Comment by ttadano** [comment]
Suppose you have carefully checked the convergence of phonon frequencies with respect to the DFT numerical parameters but still observed stable phonons. In that case, I think the qualitative difference likely originates from the different methods of phonon calculation.

**Comment by RicardoCecconello** [comment]
Thank you for the reply!
